docs/lang/articles/misc/gui.md

```diff
@@ -442,8 +247,8 @@ while gui.running:
 
 ## Image I/O
 
-[`ti.imwrite(img, filename)`](https://api-docs.taichi.graphics/src/taichi.misc.html?highlight=imwrite#taichi.misc.image.imwrite)
-can export a `np.ndarray` or Taichi field (`ti.Matrix.field`,  `ti.Vector.field`, or `ti.field`) to a specified location `filename`.
+`ti.imwrite(img, filename)` exports an `np.ndarray` or a Taichi field
```

Comment:
Maybe we don't need an `an` but an `a` for `np.ndarray`?

Reply:
It ought to be "an" here. The pronunciation of np is /'en'pi:: /, which starts with a vowel.